cluster-scripts/run-imagej.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
```

Comment:
Consider adding a brief description what this script is for.

In addition, consider renaming the file to e.g. run-imagej-server.sh.